### ISSUE TITLE
How can I define schema for each key if I have hundreds of key names

### ISSUE THREAD

**Issue by Bouncue** [sindresorhus/electron-store#76]
Say my config.json file contains hundreds of lines:
```
{
   "1": "...",
   "2": "...",
   "3": "...",
   "4": "...",
   "5": "...",
   ....,
}
```

All of values are string type. How is the quick way to define schema?

**Comment by sindresorhus** [comment]
You can just use a for-loop. It's just JS. You can dynamically construct the schema object. However, it looks like you should just use an array instead.

**Comment by Bouncue** [comment]
Thanks for your suggestion, though a bit too late.

I managed to solved this by create a function that receive two value namely the object key and the schema type, construct the schema then return it.

I can't use array, I need to store my data exactly like I have described above the keys are important indentifier.

I suggest you to remove this line in your docs:

"support all validation keywords and formats."

A bit misleading though, is that really as it says? Turns out it's not.
Things like "patternProperties, patternProperties." Are not possible to be applied in electron-store's schema.
